cogs/FamRankings.py

```diff
@@ -227,7 +227,8 @@ async def amifam(self, ctx):
             inline=True
         )
 
-        pb.generate_bar(users[f'{ctx.author.id}']['experience'], users[f'{ctx.author.id}']['rank'])
+        pb.generate_bar(users[f'{ctx.author.id}']
+                        ['experience'], users[f'{ctx.author.id}']['rank'])
```

Comment:
idk if auto formatting did this, but idk if `['experience']` should be on a separate  line since it's accessing `users[f'{ctx.author.id}']`. It won't impact anything, but hinders readability a bit

Reply:
yea, formatting did this and it shouldn't impact anything since python cares about whitespace, but I could be wrong there so I'll change it back just in case

Reply:
moved back